src/users/pages/Profile.vue

```diff
@@ -3,6 +3,7 @@
     <Profile
       :user="user"
       :current-group="currentGroup"
+      @report="report"
```

Comment:
As the ProfileUI component is now tied to `vuex`, we could merge `Profile.vue` and `ProfileUI.vue` -> later?

Not so nice for storybook though, so maybe we can entangle it again...? TrustButton is also tied to the datastore however...